README.md

```diff
@@ -0,0 +1,308 @@
+
+Something simple has a clear purpose and direction, while something easy is that which is near at hand and familiar.  Using ID's as CSS selectors is easy, but not necessarily simple, because in the long run it will create complicated specificity issues.
+
+### find styles that exist or create new styles
+
+The names of classes should reflect the component they are providing styles for so that a developer can look some markup and have a reasonable guess where to look for a style (or what to search for). In addition, the file structure of our components and stylesheets should make finding relevant styles quick. 
+
+### create or modify pages
+
+By following the guidelines here, developers should be able to create new components and styles without worrying about conflicting with existing styles.
+
+### with confidence
+
+One of the biggest frustrations with tangled CSS is that it’s nearly impossible to know when a local change has an unwanted side-effect. Developers can prevent these headaches by sticking to stricter conventions.
+
+## Our Tools
+
+Our front-end code base at Buildium consists of a series of individual Angular.js applications, each serving a different part of our product.  Each of these apps consists of even smaller components (directives, views, and controllers).  And, we store common components that are used across multiple parts of the product in a directory called 'buildium-base'.
+
+To help organize and optimize our stylesheets, we use LESS.
+
+
+## Organizing stylesheets
+
+
+A typical Angular component directory (a search bar directive, in this case) will look like this:
+
+```
+buildium-base/components/search
+
+-search.html           //template
+-search.js             //directive
+-search.less           //styles
+-search.test.js        //unit test
+
+```
+
+As you can see, this component contains it's own stylesheet.  This is imported into a file in the root components directory called main.less.
+
+components/main.less is then imported into a styles directory within buildium-base.  This style directory contains common elements like mixins, variables, and other tools that we'd like access to when writing component specific styles.  This approach keeps the component specific stylesheets lean, and also reserves a special place to write styles that can be used globally.
+
+##Common Variables
+
+Use common mixins for visual behaviors that are likely repeated throughout the site such as colors, box-shadow, border-radius, etc...
+
+All colors should be defined as common variables, and we store them in buildium-base/styles/variables/colors.less
+
+##Naming
+
+We use the Block Element Modifier (BEM) naming convention.  BEM-style names are, in general, very descriptive. They also provide a way of indicating hierarchy without the side-effects of nesting classes and causing specificity battles. Finally, BEM is very conducive to creating a component-based UI (which we strive for in our Angular-based code).
+
+block__element-modifier
```

Comment:
This is sort of just throw in the doc. Maybe some description: "Class names in BEM follow this pattern: block__element--modifier"?

Also, I think we're missing the second dash before modifier.